### DISCUSSION TITLE
How to decrease the chances of split models?

### DISCUSSION THREAD

**Discussion by yanxke** [colmap/colmap#1662] [Q&A]
I've noticed that often the scenes I capture are split into multiple models, each of which are fairly complete and seem like they cover the same areas of the scene.  Using the model_merger command fails, presumably because the matching points have already been filtered out.  Are there parameters I can tune to make the matches less strict, which can help merge the models?

**Comment by ahojnnes** [comment]
For whatever reason, colmap fails to connect the different parts of the scene into the same model due to lack of visual overlap or difficulty to match. Are you using sequential or exhaustive matching? Did you capture enough images?

**Reply by yanxke** [reply]
Yes, the image capture was pretty extensive with lots of overlap. In one experiment, I took pictures of the same scene using the phone in the portrait and landscape orientations. For some reason it produced two models, where the portrait photos were all in one model and the landscape photos were in the other model.
